templates/add_blurb.html

```diff
@@ -38,7 +38,7 @@ <h3>📜🤖 Blurb it again?</h3>
                         <select id="section" name="section" class="form-control form-inline" required>
                             <option></option>
                             <option>Security</option>
-                            <option>Core and Builtins</option>
+                            <option>Core_and_Builtins</option>
```

Comment:
Would it be good to use underscores in the option value and spaces in the human-readable text?

Reply:
```suggestion
                            <option value="Core_and_Builtins">Core and Builtins</option>
```